src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -183,6 +183,15 @@ In this job, you will connect your application code to the data stack by connect
 > - npm installed ([set up npm](https://docs.npmjs.com/getting-started))
 > - Data stack deployed
 
+### Amplify Codegen to generate the Queries,Mutations and Subscriptions 
```

Comment:
Very if this happens automatically with `push` otherwise we maybe move this into expander or link out for details.